dev-java/saxon/saxon-9.2.0.6-r3.ebuild

```diff
@@ -16,10 +16,11 @@ SLOT="9"
 KEYWORDS="amd64 ~arm arm64 ppc64 x86"
 
 CDEPEND="
-	dev-java/xom:0
-	dev-java/jdom:0
+	dev-java/ant-core:0
```

Comment:
i think to record it correctly in `/usr/share/saxon-9/package.env` we need a revbump

Reply:
> i think to record it correctly in `/usr/share/saxon-9/package.env` we need a revbump

I decided to do it wiithout revbump because there is no change in `package.env`. Before and after, in both cases it shows `BUILD_DEPEND="ant-core"`.
```
DESCRIPTION="A XSLT and XQuery Processor"
GENERATION="2"
SLOT="9"
CATEGORY="dev-java"
PVR="9.2.0.6-r3"
LIBDIR="lib64"
CLASSPATH="/usr/share/saxon-9/lib/saxon.jar"
DEPEND="dom4j-1:jdom:xom"
VM=">=virtual/jre-1.8:*"
BUILD_DEPEND="ant-core"
TARGET="1.8"
SOURCE="1.8"
MERGE_VM="openjdk-21"
MERGE_COMPILER="javac"
```

Reply:
ah, ok, i didn't know the slot is stripped. thanks for the clarification.